core/base/src/TApplication.cxx

```diff
@@ -476,9 +476,11 @@ void TApplication::GetOptions(Int_t *argc, char **argv)
          Long_t id, flags, modtime;
          char *arg = strchr(argv[i], '(');
          if (arg) *arg = '\0';
-         char *dir = gSystem->ExpandPathName(argv[i]);
-         // ROOT-9959: we do not continue if we could not expand the path
-         if (!dir) continue;
+         TString dir(argv[i]);
+         if (!gSystem->ExpandPathName(dir)) {
```

Comment:
Like the `access path name` the return value of `ExpandPathName` is inverted from the usual.
```suggestion
         if (gSystem->ExpandPathName(dir)) {
```

Reply:
Ha that's right! I already fixed this in https://github.com/root-project/root/pull/6874, but I missed to catch it here. Thanks!